docs/guides/environments.md

```diff
@@ -11,6 +11,8 @@ If no `--env` is provided, it uses the `keras` image by default, which comes wit
 | --------- | ------------------ | ------------------------ |-------------|
 | Keras | keras      | Tensorflow 1.1.0 + keras 2.0.6 on Python3.5. |  |
 |       | keras:py2  | Tensorflow 1.1.0 + keras 2.0.6 on Python2. |  |
+| Tensorflow 1.4 | tensorflow-1.4  | Tensorflow 1.4.0 + Keras 2.0.6 on Python3.6. | [floydhub/tensorflow](https://hub.docker.com/r/floydhub/tensorflow/) |
```

Comment:
@houqp can you confirm if the versions are right?

Reply:
keras version has been updated to 2.0.8